kubeflow/jupyter/jupyter.libsonnet

```diff
@@ -9,7 +9,7 @@
       "template.html": importstr "ui/default/template.html",
       "script.js": importstr "ui/default/script.js",
       "style.css": importstr "ui/default/style.css",
-      "spawner.py": importstr "ui/default/spawner.py",
+      "spawner.py": std.strReplace(importstr "ui/default/spawner.py", "\\\n", ""),
```

Comment:
Why is this needed? The JupyterHub UI seems to be working fine without this change.

Reply:
Part of in-cluster deployment of these resource if emitController==true (default=false). 
It doesn't affect normal kfctl creation/deployment.